config/bs2000-strict.conf

```diff
@@ -268,6 +271,7 @@ sequential-delimiters:			unconformable
 record-delim-with-fixed-recs:		unconformable
 record-sequential-advancing: error
 missing-statement:			warning		# not verified yet
+missing-period:				error #when format not in {fixed,free}, not verified yet
```

Comment:
that should really have a tab before the `#` and a space afterwards, if this isn't that way in 3.x please do it there, so we merge that change later :-)
[{applies to all of those conf files)

Reply:
Indeed it should also be fixed in 3.x. But actually, there are several lines where there is a single space instead of a tab between the value and the comment. So maybe this is a chance to cleanup all those lines in one go ? (maybe no need to go through a GitHub PR for this ?)

Reply:
yes and yes